src/Entity/CategoryManager.php

```diff
@@ -105,9 +105,6 @@ public function getSubCategoriesPager($categoryId, $page = 1, $limit = 25, $crit
      */
     public function getRootCategoryWithChildren(CategoryInterface $category)
     {
-        if (null === $category->getContext()) {
```

Comment:
Why did you remove this? This can still happen if you've create a category outside the admin panel

Reply:
@core23 your right, let me push ah update...

Reply:
Forget what I said, context can be null (at database level) for category. So It shouldn't happen, why you say it's possible?

Reply:
@core23 ping